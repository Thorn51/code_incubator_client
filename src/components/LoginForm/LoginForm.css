form {
  display: flex;
  flex-direction: column;
  width: 90%;
  margin: 50px auto 10px auto;
  padding: 15px;
}

input[type="text"],
input[type="password"],
input[type="url"],
textarea,
select {
  width: 100%;
  padding: 10px;
  margin: 0 0 10px 0;
  background: #f1f1f1;
}

input[type="text"]:focus,
input[type="password"]:focus,
input[type="url"]:focus,
textarea:focus {
  background-color: #ddd;
  outline: none;
}

hr {
  border: 1px solid #f1f1f1;
  margin-bottom: 15px;
}

.form_button {
  background-color: black;
  color: white;
  padding: 16px 20px;
  margin: 8px 0;
  border: none;
  cursor: pointer;
  width: 100%;
  opacity: 0.9;
}

.form_redirect {
  width: 80%;
  margin-left: auto;
  margin-right: auto;
}

/* LOGIN FORM */

.login_register {
  width: 80%;
  margin-left: auto;
  margin-right: auto;
}
