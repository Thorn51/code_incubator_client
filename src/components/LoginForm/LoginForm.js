import React from "react";
import "./LoginForm.css";

export default class LoginForm extends React.Component {
  render() {
    return null;
  }
}
