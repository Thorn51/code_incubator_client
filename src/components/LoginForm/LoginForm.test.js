import React from "react";
import ReactDOM from "react-dom";
import renderer from "react-test-renderer";
import LoginForm from "./LoginForm";

describe("<LoginForm />", () => {
  it("renders without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(<LoginForm />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it("renders as expected", () => {
    const tree = renderer.create(<LoginForm />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
