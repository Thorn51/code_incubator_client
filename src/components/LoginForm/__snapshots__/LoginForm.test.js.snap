// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoginForm /> renders as expected 1`] = `
<form
  className="registration_form"
  onSubmit={[Function]}
>
  <h2
    className="section_title"
  >
    Login
  </h2>
  <hr />
  <label
    htmlFor="email"
  >
    Email:
  </label>
  <input
    id="email"
    name="email"
    placeholder="Enter Email"
    required={true}
    type="text"
  />
  <label
    htmlFor="password"
  >
    Password:
  </label>
  <input
    id="password"
    name="password"
    placeholder="Enter Password"
    required={true}
    type="password"
  />
  <div
    role="alert"
  />
  <button
    className="form_button"
    type="submit"
  >
    Login
  </button>
</form>
`;
